lnbits/db.py

```diff
@@ -173,16 +173,26 @@ async def fetch_page(
         values: Optional[List[str]] = None,
         filters: Optional[Filters] = None,
         model: Optional[Type[TRowModel]] = None,
+        group_by: Optional[str] = None,
     ) -> Page[TRowModel]:
         if not filters:
             filters = Filters()
         clause = filters.where(where)
         parsed_values = filters.values(values)
 
+        if group_by:
+            assert re.fullmatch(
+                r"[a-zA-Z_][a-zA-Z0-9_]*(\.[a-zA-Z_][a-zA-Z0-9_]*)?", group_by
+            ), "GROUP BY regex failed"
+            group_by = f"GROUP BY {group_by}"
+        else:
+            group_by = ""
```

Comment:
nitpick: 
 - if the param declaration is `group_by: Optional[str] = "",` then this `else` is not required anymore